cachito/workers/config.py

```diff
@@ -163,7 +163,7 @@ def configure_celery(celery_app):
         # a hack taken from flask.config.from_pyfile.
         _user_config = {}
         with open(prod_config_file_path, mode="rb") as config_file:
-            exec(compile(config_file.read(), prod_config_file_path, "exec"), _user_config)
+            exec(compile(config_file.read(), prod_config_file_path, "exec"), _user_config)  # nosec
```

Comment:
I will be glad to see a comment on why we want `bandit` to ignore this line. And maybe a comment for one of the `password` lines.